packages/node/test/manual/release-health/single-session/errors-in-session-capped-to-one.js

```diff
@@ -0,0 +1,65 @@
+const Sentry = require('../../../../dist');
+const { assertSessions, constructStrippedSessionObject, BaseDummyTransport } = require('../test-utils');
+let sessionCounter = 0;
+process.on('exit', ()=> {
+  if (process.exitCode !== 1) {
+    console.log('SUCCESS: All application mode sessions were sent to node transport as expected');
```

Comment:
This test doesn't actually verify the behavior. There's nothing stopping it from passing if only a single session is sent as you never assert on `sessionCounter` count.